feathr_project/test/test_azure_spark_e2e.py

```diff
@@ -443,17 +465,54 @@ def test_feathr_materialize_with_time_partition_pattern():
                                        feature_names=[
                                            "f_loc_avg_output", "f_loc_max_output"],
                                        backfill_time=backfill_time_tpp)
-    client_consumer.materialize_features(settings_tpp, allow_materialize_non_agg_feature=True)
+    client_consumer.materialize_features(settings_tpp)
     # assuming the job can successfully run; otherwise it will throw exception
     client_consumer.wait_job_to_finish(timeout_sec=Constants.SPARK_JOB_TIMEOUT_SECONDS)
 
     # download result and just assert the returned result is not empty
     # by default, it will write to a folder appended with date
-    res_df = get_result_df(client_consumer, "avro", output_path_tpp + "/df0/daily/2020/05/20")
+    res_df = get_result_df(client_consumer, "avro", output_path_tpp + "/df0/daily/2020/05/21")
+    assert res_df.shape[0] > 0
+   
+@pytest.mark.skip(reason="Will add back when updated scala jar is applied")    
```

Comment:
Why are we skipping here?

Reply:
Because our current scala jar doesn't support this 'postfix_path' param.